package.json

```diff
@@ -21,17 +21,23 @@
     "nunjucks": "^3.2.4"
   },
   "devDependencies": {
-    "@eslint/js": "^9.0.0",
+    "@eslint/js": "^9.9.1",
     "@types/axios": "^0.14.0",
+    "@types/eslint__js": "^8.42.3",
     "@types/express": "^4.17.21",
     "@types/express-session": "^1.18.0",
     "@types/node": "^20.16.1",
     "@types/nunjucks": "^3.2.6",
+    "dotenv": "^16.4.5",
     "eslint": "^8.57.0",
+    "eslint-config-standard": "^17.1.0",
+    "eslint-plugin-import": "^2.29.1",
```

Comment:
Do you need these plugins as I don’t see them being used by the config file

Reply:
No I don't, they must have been remnants from the attempt at google linting. Removed them in the most recent commit.

Reply:
There are still 2 in there, are those needed

Reply:
They've been removed